pkg/boot/syslinux/syslinux_test.go

```diff
@@ -268,6 +268,49 @@ func TestAppendFile(t *testing.T) {
 				},
 			},
 		},
+		{
+			desc:          "valid config with two Entries, and a nerfdefault override",
+			configFileURI: "pxelinux.cfg/default",
+			schemeFunc: func() curl.Schemes {
+				s := make(curl.Schemes)
+				fs := curl.NewMockScheme("tftp")
+				fs.Add("1.2.3.4", "/foobar/pxelinux.0", "")
+				conf := `default foo
+
+				nerfdefault bar
```

Comment:
Note that the order of these matters. Do you want it to?

Reply:
Fair point. I have modified the code to support any ordering of these labels. PTAL, thank you.